### PR TITLE
[TH2-4068] Add ability to declare keep alive timeout option for grpc client

### PR DESCRIPTION
**Pull request by isengrims**

### CODE REVIEW

File: src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt [hunk 1]
```diff
@@ -31,7 +31,8 @@ data class GrpcServiceConfiguration(
     @Deprecated("For removal since v3.37") @JsonProperty(required = true) var strategy: RoutingStrategy<*>,
     @JsonProperty(required = true, value = "service-class") var serviceClass: Class<*>,
     @JsonProperty(required = true) var endpoints: Map<String, GrpcEndpointConfiguration> = emptyMap(),
-    @JsonProperty var filters: List<Filter> = emptyList()
+    @JsonProperty var filters: List<Filter> = emptyList(),
+    @JsonProperty(required = false) var keepAliveInterval: Long = 300L
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt:35]: Let's move this property to GrpcConfiguration.client.keepAliveInterval

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]
Please update README:
* Release notes
* grpc configuration
Bump minor version

File: src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt (hunk 1 above)
Comment (isengrims) [reply src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt]: Done

File: gradle.properties [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-release_version=3.40.0
+release_version=3.40.1
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle.properties:16]:
```suggestion
release_version=3.41.0
```

File: README.md [hunk 3]
```diff
@@ -309,6 +309,9 @@ dependencies {
 
 ## Release notes
 
+### 3.40.1 
++ gRPC configuration now contains `client` property where `keepAliveInterval` can be defined for client grpc services. (default: 300 sec)
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:313]: Change the default value to 60 sec
```suggestion
+ gRPC configuration now contains `client` property where `keepAliveInterval` can be defined for client grpc services. (default: 60 sec)
```

File: README.md [hunk 4]
```diff
@@ -309,6 +309,9 @@ dependencies {
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:309]: Please information about grpc_router.json configuration

File: src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt [hunk 5]
```diff
@@ -38,6 +39,10 @@ data class Filter(
     @JsonProperty(required = true) var properties: List<FieldFilterConfiguration>,
 ) : Configuration()
 
+data class GrpcClientConfiguration(
+    @JsonProperty var keepAliveInterval: Long = 300L
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt:43]:
```suggestion
    @JsonProperty var keepAliveInterval: Long = 60L
```

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -309,6 +348,9 @@ dependencies {
 
 ## Release notes
 
+### 3.40.1
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:351]:
```suggestion
### 3.41.0 
```

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

**Pull request merged**